terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -2,6 +2,18 @@ variable "data_share_processor_name" {
   type = string
 }
 
+variable "peer_share_processor_aws_account_id" {
+  type = string
+}
+
+variable "peer_share_processor_manifest_domain" {
+  type = string
+}
+
+variable "peer_validation_bucket_name" {
+  type = string
+}
```

Comment:
Boy I wish I hadn't made these separate modules. Plumbing all these input variables is just a chore. Nothing to be done about it in this change, but I filed #117 to clean this up.